src/CoreWCF.Primitives/src/CoreWCF/Channels/Message.cs

```diff
@@ -284,7 +284,7 @@ public static Message CreateMessage(MessageVersion version, MessageFault fault,
             return new BodyWriterMessage(version, action, new FaultBodyWriter(fault, version.Envelope));
         }
 
-        internal Exception CreateMessageDisposedException()
+        public Exception CreateMessageDisposedException()
         {
```

Comment:
This doesn't need any state from the Message. Leave internal and make a static helper method in WebHttp

Reply:
This has been addressed.